app/models/book.rb

```diff
@@ -0,0 +1,2 @@
+class Book < ActiveRecord::Base
```

Comment:
I didn't both adding any Rails validations here since we don't have any user input. There'ss a DB constrains to prevent nil titles.